src/http-gateways/path-gateway.md

```diff
@@ -151,8 +151,8 @@ content type parameters.
 For example:
 
 - [application/vnd.ipld.raw](https://www.iana.org/assignments/media-types/application/vnd.ipld.raw) – disables [IPLD codec deserialization](https://ipld.io/docs/codecs/), requests a verifiable raw [block](https://docs.ipfs.io/concepts/glossary/#block) to be returned
-- [application/vnd.ipld.car](https://www.iana.org/assignments/media-types/application/vnd.ipld.car) – disables [IPLD codec deserialization](https://ipld.io/docs/codecs/), requests a verifiable [CAR](https://docs.ipfs.io/concepts/glossary/#car) stream to be returned
-- [application/x-tar](https://en.wikipedia.org/wiki/Tar_(computing)) – returns UnixFS tree (files and directories) as a [TAR](https://en.wikipedia.org/wiki/Tar_(computing)) stream. Returned tree starts at a root item which name is the same as the requested CID. Produces 400 Bad Request for content that is not UnixFS.
+- [application/vnd.ipld.car](https://www.iana.org/assignments/media-types/application/vnd.ipld.car) – disables [IPLD codec deserialization](https://ipld.io/docs/codecs/), requests a verifiable [CAR](https://docs.ipfs.io/concepts/glossary/#car) stream to be returned with implicit or explicit [`dag-scope`](https://specs.ipfs.tech/http-gateways/trustless-gateway/#dag-scope-request-query-parameter) for blocks at the terminus of the specified path and the blocks required to traverse path segments from root CID to the terminus.
```

Comment:
I find this line and how `dag-scope` influences it to be a bit confusing. 

- If you don't pass an explicit `dag-scope` it will default to `all`, returning the terminus block the "entire contiguous DAG that begins at the end of the path query". But what is meant by that?

For example: if you request `application/vnd.ipld.car` with the path `bafy...jomu/assets/logo.png` in the following DAG
![image](https://github.com/user-attachments/assets/95a7a6d4-b3a7-432e-aecd-2308eb44c951)


How will dag-scope influence the response?
- **`all`**: root block `bafy...jomu`, `assets` dir block `bafy...idcm`, logo.png block `bafy...wwoi` and raw data blocks. 
- `block`: This would need the same blocks as `all` in order to "verify the specified path segments."

What am I misunderstanding?